doc/tutorials/basic/widgets.md

```diff
@@ -0,0 +1,198 @@
+:::
+
+:::{note}
+For an in-depth understanding of how a *widget* like `Button` works, please refer to its *reference guide*.
+:::
+
+Click [this link](../../reference/widgets/Button.ipynb) to the `Button` Reference Guide, and let's spend a few minutes familiarizing ourselves with its organization and content.
+
+It should look like
+
+[![Button reference guide](../../_static/images/widgets_button_reference.png)](../../reference/widgets/Button.ipynb)
+
+## Accept Text
+
+Now, let's run the code below together:
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+pn.widgets.TextInput(
+    description="The text given to the AI",
+    disabled=True,
+    max_length=15,
+    name="Prompt",
+    placeholder="What is Python?",
+).servable()
+```
+
+Feel free to hover over the circle with the question mark inside. You will see the `description` displayed as a tooltip.
+
+If you are in a notebook or serving as an app, try:
+
+- Enabling the `TextInput` by changing `disabled=True` to `disabled=False`.
+- Entering the text `1234567890123456`. You cannot because the `max_length` is `15`. The final `6` cannot be entered.
+
+:::{admonition}
+Your success with Panel hinges on your ability to navigate the [Component Gallery](../../reference/index.md) and utilize the *reference guides* effectively.
+:::
+
+Click [this link](../../reference/index.md#widgets) to access the Widgets Section of the [Component Gallery](../../reference/index.md). Locate the [TextInput Reference Guide](../../reference/widgets/TextInput.ipynb) and open it. You don't have to delve into the details right now.
+
+It should look like
+
+[![Widgets Gallery and TextInput Reference Guide](../../_static/images/widgets_textinput_reference.png)](../../reference/index.md#widgets)
+
+## Accept an object from a list
+
+Let's run the code below together:
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+pn.widgets.Select(
+    description="Select a Study",
+    name="Study",
+    options=["Biology", "Chemistry", "Physics"],
+).servable()
+```
+
+:::{note}
+Panels widgets come in groups of *interchangeable* widgets. For example, it's very easy to replace the `Select` widget with the `RadioButtonGroup` widget.
+:::
+
+Now, run the code below:
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+pn.widgets.RadioButtonGroup(
+    description="Select a Study",
+    name="Study",
+    options=["Biology", "Chemistry", "Physics"],
+).servable()
+```
+
+:::{Admonition} Note
+Usually, widgets allow us to specify the *initial* `value`.
+:::
+
+Now, let's run the code below:
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+pn.widgets.RadioButtonGroup(
+    button_type="primary",
+    button_style="outline",
+    description="Select a Study",
+    name="Study",
+    options=["Biology", "Chemistry", "Physics"],
+    value="Physics",
+).servable()
+```
+
+You will notice how `Physics` is now displayed as the *initial* `value`, and how the `Button`'s type and style were changed.
+
```

Comment:
I feel like this tutorial should show what widgets are useful for, either link to a bind page or show an example in a note.

Reply:
I understand your concern. I'm also concerned about it.

My approach right now is to focus on displaying widgets including learning to use basic arguments. Then the next, separate tutorial is about using widgets via `pn.bind`. I do mention this in the intro section.

I do this because then users can take one instead of two steps at the time.

I don't know what will be most useful for users. We might not know before this meets the *basic user*.

What would you recommend we do @ahuang11, @philippjfr  and others?

Reply:
A quick example of bind and then a link to learn more.